<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductDialog</class>
 <widget class="QDialog" name="AddProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Product</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/rotable/db_add.png</normaloff>:/rotable/db_add.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="_labelInfo">
     <property name="text">
      <string>Please specify the product you want to add:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="_labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="_lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_lineEditAmount"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_labelAmount">
       <property name="text">
        <string>Info:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="_lineEditInfo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_labelPrice">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="_spinBoxPrice">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_labelPriceString">
         <property name="text">
          <string>0,00 â‚¬</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_labelIcon">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="_comboBoxIcons"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="_labelCategory">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="_comboBoxCategories"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_lineEditName</tabstop>
  <tabstop>_lineEditAmount</tabstop>
  <tabstop>_lineEditInfo</tabstop>
  <tabstop>_spinBoxPrice</tabstop>
  <tabstop>_comboBoxIcons</tabstop>
  <tabstop>_comboBoxCategories</tabstop>
  <tabstop>_pushButtonCancel</tabstop>
  <tabstop>_pushButtonAdd</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
